static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -214,7 +214,7 @@ const makeEpicABTestVariant = (
         }`,
         campaignCode,
         supportURL: addTrackingCodesToUrl({
-            base: supportContributeURL(),
+            base: initVariant.supportBaseURL || supportContributeURL(),
```

Comment:
pass this through `addCountryGroupToSupportLink()`

it will (should!) leave it unchanged if it includes a country code. if not, it will try and add it if it already has a geolocation stored